src/models/__tests__/Collaborator.spec.ts

```diff
@@ -1,5 +1,23 @@
 import casual from 'casual';
 import { Collaborator, TemplateCollaborator } from "../Collaborator";
+import { Template } from '../Template';
```

Comment:
Not that urgent, but there are tests missing for findByTemplateId, findById, findByEmail, and findByTemplateIdAndEmail that we might want to add.

Reply:
Yeah, I went back and forth on that since they're not really doing much aside from defining the SQL statement and passing it off to the generic MySqlModel functions